framework/source/class/qx/test/dev/unit/Sinon.js

```diff
@@ -235,9 +235,6 @@ qx.Class.define("qx.test.dev.unit.Sinon",
       var func = function() {};
       var obj = {"a": function() {}};
 
-      var spy = this.spy(func);
-      var stub = this.stub(obj, "a");
-      var xhr = this.useFakeXMLHttpRequest();
```

Comment:
What if this lines test Sinon itself? Of course the variable ```spy``` is not used anymore, but the call ```this.spy(func)``` itself could be the test. If ```this.spy(func)``` fails because of an exception, then the Sinon integration may be faulty.